src/Reflection/Adapter/ReflectionClass.php

```diff
@@ -184,7 +184,9 @@ public function hasProperty($name)
     }
 
     /**
-     * {@inheritDoc}
+     * @param string $name
```

Comment:
We shouldn't change the documented API in the `Adapter` ns

Reply:
You already have, though - documented [here](https://github.com/Roave/BetterReflection/issues/389)

Reply:
Yeah, but that bug is being fixed. This needs revert or we can wait for the other patch to go in before reverting

Reply:
Reverted